Project.toml

```
@@ -10,7 +10,7 @@ JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 [compat]
 HTTP = "1"
 JSON = "0.21"
-julia = "1"
+julia = "1.11"
```

```suggestion
julia = "1.10"
```
We need to support LTS. You can activate the folder with `Pkg.activate(".")` and then `Pkg.add("ScopedValues")`. That way you can `using ScopedValues` instead of the one in Base.